src/unitxt/metrics.py

```diff
@@ -63,6 +63,7 @@ class MetricWithConfidenceInterval(Metric):
     # Use None to disable confidence interval computation.
     n_resamples: int = None
     confidence_level: float = 0.95
+    ci_scores: List[str] = None
```

Comment:
I like that there is now a way to define which scores to run on.  I suggest, that by default it will be set to an empty list (so by default, ci will not be calculated).  Then, we will add for each metric the relevant scores or the default score.

Reply:
I opened an issue for that: https://github.com/IBM/unitxt/issues/432.

Reply:
@matanor @assaftibm .  Do we want to make the change in this PR or in separate one?

Reply:
@assaftibm @yoavkatz i tend towards having a separate one

Reply:
OK. Let's have a separate one.  There are few failing tests that need to be addressed.

Reply:
@yoavkatz I guess I'm not seeing the full picture, but are there metrics for which CI won't be computed? (and if so, why is that?)

Reply:
The reason is runtime. For example, it was disabled for the default version of rouge ([here](https://github.com/IBM/unitxt/blob/main/prepare/metrics/rouge.py#L5)). 

There are other cases of users that asked to turn it off. 

This issue is mainly (or completely) for global metrics.